elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/impl/ElasticsearchFlowStage.scala

```diff
@@ -19,7 +18,9 @@ import org.elasticsearch.client.{Response, ResponseListener, RestClient}
 import scala.collection.immutable
 
 /**
- * INTERNAL API
+ * INTERNAL API.
+ *
+ * Updates Elasticsearch without any built-in retry logic.
  */
 @InternalApi
 private[elasticsearch] final class ElasticsearchFlowStage[T, C](
```

Comment:
The history would be clearer if you didn't rename the `ElasticSearchSimpleFlowStage`.